src/Application/Actions/Person/SetFirstPassword.php

```diff
@@ -12,12 +12,11 @@
 use BigGive\Identity\Repository\EmailVerificationTokenRepository;
 use BigGive\Identity\Repository\PersonRepository;
 use Laminas\Diactoros\Response\JsonResponse;
-use OpenApi\Annotations as OA;
+use BigGive\Identity\Client;
```

Comment:
```suggestion
use OpenApi\Annotations as OA;
use BigGive\Identity\Client;
```
I think IntelliJ IDEs prefer us to keep this when using the comment annotations?